CustomeScene class

hectorsvill opened the pull request:
Just added spriteKit CustomeScene class

mosesrobinson commented:
Hector, great work! I'm glad to see your strong use of MVC. You went above and beyond and refactored the project and removed the shared instance.

Even though, I had a Singleton solution to go off of, it was very easy to read your code and understanding what you did. The amount of commits you made is outstanding. 

My suggestion would be to write full sentences as to what you are doing or what you did, making it very clear to anyone jumping in to your codebase able to understand. Example: instead of "project clean up." Be specific as to what that clean up was, no matter how trivial it may sound, or be. 

Same goes for using "sc" for settingController. It is always best to be explicit about what you're creating a variable for. Especially with the amount of times "sc" was used in this project.

hectorsvill commented:
> Hector, great work! I'm glad to see your strong use of MVC. You went above and beyond and refactored the project and removed the shared instance.
> 
> Even though, I had a Singleton solution to go off of, it was very easy to read your code and understanding what you did. The amount of commits you made is outstanding.
> 
> My suggestion would be to write full sentences as to what you are doing or what you did, making it very clear to anyone jumping in to your codebase able to understand. Example: instead of "project clean up." Be specific as to what that clean up was, no matter how trivial it may sound, or be.
> 
> Same goes for using "sc" for settingController. It is always best to be explicit about what you're creating a variable for. Especially with the amount of times "sc" was used in this project.

Thank you For y

PaulSolt commented on Contained-iOS6/Contained-iOS6/Settings.swift:
```diff
@@ -13,7 +13,7 @@ class Settings {
 	var shouldZoom = false
 	var happyCrab = true
-	var zoom = false
+	var shouldFade = false
```
Nice variable name!
